WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -100,17 +103,27 @@ public void onClick(View v) {
                     fabMenu.collapse();
                 }
             });
+            mFabView = fabMenu;
         } else {
-            fabMenu.setVisibility(View.GONE);
             FloatingActionButton fabMenuAddDotOrg = (FloatingActionButton) findViewById(R.id.fab_add_dotorg);
-            fabMenuAddDotOrg.setVisibility(View.VISIBLE);
             fabMenuAddDotOrg.setOnClickListener(new View.OnClickListener() {
                 @Override
                 public void onClick(View v) {
                     ActivityLauncher.addSelfHostedSiteForResult(SitePickerActivity.this);
                 }
             });
+            mFabView = fabMenuAddDotOrg;
         }
+
+        // animate fab in after brief delay (to give time for activity transition to complete first)
+        new Handler().postDelayed(new Runnable() {
+            @Override
+            public void run() {
+                if (!isFinishing()) {
+                    ReaderAnim.showFab(mFabView, true);
+                }
+            }
+        }, 500);
```

Comment:
I think we should throw the 500 into a const variable. Also, I looked into [the animation](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/res/anim/activity_slide_in_from_left.xml) for the site picker activity it looks like it uses 200ms to transition. Maybe we could reduce this delay to ~250ms.